src/launcher/standalone_launcher/.env.development

```diff
@@ -0,0 +1,9 @@
+REACT_APP_BROKER_HOST=web-dev.adaptivecluster.com
+REACT_APP_BROKER_PORT=80
+REACT_APP_SYMPHONY_APP_ID=reactiveTrader-dev
+ENVIRONMENT_NAME=local
+
+# The REACT_APP_ANALYTICS_URL var is only used from the OF launcher, so most of the times we'll want to 
+# launch the live app even when doing dev work on the launcher itself. Otherwise switch to the localhost one
+REACT_APP_ANALYTICS_URL=https://demo-reactive-analytics.adaptivecluster.com/
+# REACT_APP_ANALYTICS_URL=http://localhost:3005
```

Comment:
Add newline at end of file. (Please set up your IDE to insert these automatically).